packages/tenant-process/test/getTenantById.test.ts

```diff
@@ -23,7 +23,7 @@ describe("getTenantById", () => {
     await addOneTenant(tenant1);
     await addOneTenant(tenant2);
     await addOneTenant(tenant3);
-    const returnedTenant = await tenantService.getTenantById(
+    const { data: returnedTenant } = await tenantService.getTenantById(
```

Comment:
As done for attributes/delegations, I would update this test to test the entire result object including metadata

See https://github.com/pagopa/interop-be-monorepo/pull/1849/files#diff-2ab69d62d3cac341a38dac792805f097aee52ca413a8806d8b976398bf47d1afR82